src/parser/xasl_generation.c

```diff
@@ -4179,7 +4179,7 @@ pt_to_aggregate_node (PARSER_CONTEXT * parser, PT_NODE * tree, void *arg, int *c
       aggregate_list->flag_agg_optimize = false;
       BTID_SET_NULL (&aggregate_list->btid);
       if (info->flag_agg_optimize
-	  && (aggregate_list->function == PT_COUNT_STAR || aggregate_list->function == PT_COUNT
+	  && (aggregate_list->function == PT_COUNT_STAR
```

Comment:
PT_COUNT is mentioned in line 4188.
Is there a reason why you removed the PT_COUNT that was in the existing code from this line location?

Reply:
PT_COUNT is not considered and PT_COUNT will be handled in the future. So, I remove the PT_COUNT in this point.

Reply:
It would be better to treat PT_COUNT as annotation in line 4188.

if (aggregate_list->function == PT_COUNT_STAR /* || aggregate_list->function == PT_COUNT */)

Reply:
Ok, I'm gonna fix it.